bigtable/client/instance_admin_test.cc

```diff
@@ -930,3 +930,165 @@ TEST_F(InstanceAdminTest, DeleteClusterRecoverableError) {
                             "exceptions are disabled");
 #endif  // GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
 }
+
+/// @test Verify that `bigtable::InstanceAdmin::CreateCluster` works.
+TEST_F(InstanceAdminTest, CreateCluster) {
+  using ::testing::_;
+  using ::testing::Invoke;
+
+  bigtable::InstanceAdmin tested(client_);
+  EXPECT_CALL(*client_, CreateCluster(_, _, _))
+      .WillOnce(Invoke([](grpc::ClientContext*,
+                          btproto::CreateClusterRequest const& request,
+                          google::longrunning::Operation* response) {
+        auto const project_name =
+            "projects/" + kProjectId + "/instances/test-instance";
+        EXPECT_EQ(project_name, request.parent());
+        return grpc::Status::OK;
+      }));
+
+  std::string expected_text = R"(
+name: 'projects/my-project/instances/test-instance'
+location: 'Location1'
+default_storage_type: SSD
+)";
+  btproto::Cluster expected;
+  ASSERT_TRUE(
+      google::protobuf::TextFormat::ParseFromString(expected_text, &expected));
+  EXPECT_CALL(*client_, GetOperation(_, _, _))
+      .WillOnce(Invoke([](grpc::ClientContext*,
```

Comment:
Consider storing the lambda for these two first cases in a variable, and reduce the clutter a bit.